packages/cockroach/buildinfo.json

```diff
@@ -1,9 +1,8 @@
 {
-  "requires": ["ncurses"],
   "single_source" : {
       "kind": "url_extract",
-      "url": "https://binaries.cockroachdb.com/cockroach-v2.0.7.src.tgz",
-      "sha1": "8b3940ef9ab46646b4178bc93c9904b57d9f3c50"
+      "url": "https://binaries.cockroachdb.com/cockroach-v2.1.7.linux-amd64.tgz",
```

Comment:
Is this going to be statically linked against any library that isn't shipped with DC/OS in `/opt/mesosphere`?

Reply:
Maybe I don't understand the question, but why would it matter?

Reply:
We have to understand what exact dependencies are we expecting to be installed on the host system and document them - https://docs.mesosphere.com/1.13/installing/production/system-requirements/#software-prerequisites

The reason why we ship almost every dependency with DC/OS is that we can control software versions that we use to run DC/OS and test such configuration. If you're deciding to rely on OS provided library dependency it should be clear which versions we're going to test with.

Reply:
```
ldd dependencies of CockroachDB 2.0.7 (previous, self-built binary)
	linux-vdso.so.1 =>  (0x00007ffe0f5f1000)
	librt.so.1 => /lib64/librt.so.1 (0x00007fe526549000)
	libpthread.so.0 => /lib64/libpthread.so.0 (0x00007fe52632d000)
	libm.so.6 => /lib64/libm.so.6 (0x00007fe52602b000)
	libc.so.6 => /lib64/libc.so.6 (0x00007fe525c5e000)
	/lib64/ld-linux-x86-64.so.2 (0x00007fe526751000)
```
```
ldd dependencies of CockroachDB 2.1.7 (downloaded binary)
	linux-vdso.so.1 (0x00007ffe39de7000)
	librt.so.1 => /lib64/librt.so.1 (0x00007f30b286a000)
	libpthread.so.0 => /lib64/libpthread.so.0 (0x00007f30b264b000)
	libm.so.6 => /lib64/libm.so.6 (0x00007f30b22b7000)
	libc.so.6 => /lib64/libc.so.6 (0x00007f30b1ef8000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f30b2a72000)
```

Reply:
Nice, this can be closed.